tests/Node/ModuleTest.php

```diff
@@ -186,7 +186,9 @@ protected function doDisplay(array \$context, array \$blocks = []): iterable
         \$macros["macro"] = \$this->macros["macro"] = \$this->loadTemplate("foo.twig", "foo.twig", 2)->unwrap();
         // line 1
         \$this->parent = \$this->loadTemplate("layout.twig", "foo.twig", 1);
-        yield from \$this->parent->unwrap()->yield(\$context, array_merge(\$this->blocks, \$blocks));
+        if (\$this->parent) {
```

Comment:
`$this->parent` always exists as `loadTemplate` always returns an object or throws an exception.

Reply:
You're right. I just saw that too.